sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests.WebApp/OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.WebApp.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <!-- 
+  Integration tests evaluate an app's components on a broader level than unit tests. 
```

Comment:
curious: do we need this?

Reply:
Are you asking about the comment or the project?

I included the comment because this is an uncommon project type.

Reply:
The comment - it seems to be out of scope for this project (e.g. we don't try to define/clarify what integration test is in this project).